sources/libraries/bid/c_convert/bid_convert_c.c

```diff
@@ -0,0 +1,50 @@
+#include <bid/bid.h>
+#include <bid/c_convert/bid_convert_c.h>
+#include <bid/c_convert/bid_convert_errors.h>
+#include <bid/c_convert/bid_convert_header.h>
+#include <bid/c_convert/bid_convert_message_type.h>
+#include <bid/c_convert/bid_convert_method.h>
+#include <bid/c_convert/bid_convert_type.h>
+#include <bid/c_convert/bid_convert_typedef.h>
+#include <bid/parsing/bid_ast.h>
+#include <brutal/alloc.h>
+#include <brutal/base.h>
+#include <brutal/log.h>
+#include "brutal/text/str.h"
+
```

Comment:
You should include <brutal/text.h> and fix you IDE please, we already talk about that in previous PR